cmd/soroban-cli/src/cli.rs

```diff
@@ -102,6 +102,9 @@ fn set_env_from_config() {
     if let Ok(config) = Config::new() {
         set_env_value_from_config("STELLAR_ACCOUNT", config.defaults.identity);
         set_env_value_from_config("STELLAR_NETWORK", config.defaults.network);
+        if let Some(inclusion_fee) = config.defaults.inclusion_fee {
+            set_env_value_from_config("STELLAR_INCLUSION_FEE", Some(inclusion_fee.to_string()));
+        }
```

Comment:
I think we can change the `set_env_value_from_config` to something like this instead:

```rust
fn set_env_value_from_config<T: std::fmt::Display>(name: &str, value: Option<T>) {
    let Some(value) = value else {
        return;
    };

    std::env::remove_var(format!("{name}_SOURCE"));

    if std::env::var(name).is_err() {
        std::env::set_var(name, value.to_string());
        std::env::set_var(format!("{name}_SOURCE"), "use");
    }
}
```